content/en/monitors/status/status_page.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Monitor Status Page
+description: "Get an overview of your monitor status over time"
+code_lang: status_page
+type: multi-code-lang
+weight: 1
+aliases:
+- /monitors/monitor_status/
+- /monitors/manage/status
+further_reading:
+- link: "/monitors/status/graphs/"
+  tag: "Documentation"
+  text: "Learn more about monitor graphs"
+- link: "/monitors/status/events/"
+  tag: "Documentation"
+  text: "Learn more about event details"
+---
+
+<div class="alert alert-warning">The provisional status page has limited support for monitors and their features. For more details, see <a href="#restrictions-of-provisional-status-page">Restrictions of provisional status page</a>.<br><br>If you are using the legacy status page, see the <a href="/monitors/status/status_legacy">Status Page (Legacy)</a> documentation</div>
+
+## Overview
+
+When a monitor alerts, the first place to start your investigation is the monitor status page. This page displays graphs and events to show you why your monitor is in an alert status, so you can understand what is happening around a monitor alert. Further, you can access quick actions to help you move the investigation towards resolution. Use the monitor status page to:
+
+- Review all the context you need to start an investigation
+- Investigate the possible cause of an alert
+- Take action to escalate, maintain, or resolve your investigation
+
+## Header
+
+{{< img src="/monitors/status/status_page_header.png" alt="Monitor status page header" style="width:100%;" >}}
+
+The header contains the monitor's metadata, including:
+- Alert status
+- Monitor status
+- Monitor type
+- Creation date
+- Author
+- Associated teams (if available)
+- Tags
+
+On the right, you'll find the **Edit**, **Clone**, **Export**, and **Permissions** buttons.
+
+From the header, you can resolve the monitor. Resolving from the header resolves all groups in the alert and sets the monitor status to `OK` (all groups). The `resolve` function temporarily changes the monitor status to `OK` for its next evaluation, but the next evaluation proceeds as normal based on current data. To resolve from the Event details, see [Status Events][1].
+
+Additionally, view and manage downtimes impacting this monitor and create new ones to mute notifications. 
```

Comment:
Maybe here we could add a screenshot like the one below to clarify how to check donwtimes impacting this monitor
<img width="1918" alt="image" src="https://github.com/user-attachments/assets/8cfb6860-0f66-4677-a9ae-ade4d5739cbe">
()